guidelines/sc/22/focus-appearance.html

```diff
@@ -7,11 +7,11 @@ <h4>Focus Appearance</h4>
     
     <p>When a <a>user interface component</a> has keyboard focus, one or both of the following is true:</p>
```

Comment:
This, as AGWG had voted and agreed on in [February 22nd](w3.org/2022/02/22-ag-minutes#item06)

```suggestion
    <p>When the keyboard focus indicator of a <a>user interface component</a> is visible, one or both of the following is true:</p>
```

Reply:
I'm fairly sure that was superseded, I'll look it up.

Reply:
Yeah, I think you're right. it's just a result of this branch not being updated when other changes were made, I think.

Reply:
Tracing forward from Feb, the start was changed when we accepted the "[reboot](https://docs.google.com/document/d/1TI_DsJjfg9RW_A9C1XfqgtfYqNMfnxeCz6lxjpOQ9rk/edit#heading=h.tcutgbbczlhr)" version.